### PR TITLE
Fix regex pattern for exception thrown in tests

### PR DESCRIPTION
**Pull request by mlxd**
### Before submitting

Please complete the following checklist when submitting a PR:

- [x] All new features must include a unit test.
      If you've fixed a bug or added code that should be tested, add a test to the
      [`tests`](../tests) directory!

- [x] All new functions and code must be clearly commented and documented.
      If you do make documentation changes, make sure that the docs build and
      render correctly by running `make docs`.

- [x] Ensure that the test suite passes, by running `make test`.

- [x] Add a new entry to the `.github/CHANGELOG.md` file, summarizing the
      change, and including a link back to the PR.

- [x] Ensure that code is properly formatted by running `make format`. 

When all the above are checked, delete everything above the dashed
line and fill in the pull request template.

------------------------------------------------------------------------------------------------------------

**Context:** This updates a given test to match a new exception string thrown by PennyLane.

**Description of the Change:** As above.

**Benefits:** Ensures all tests pass.

**Possible Drawbacks:** None

**Related GitHub Issues:**

### CODE REVIEW

File: .github/CHANGELOG.md [hunk 1]
```diff
@@ -16,6 +16,9 @@
 
 ### Bug fixes
 
+* Test updates to reflect new additions to PennyLane.
+[(#318)](https://github.com/PennyLaneAI/pennylane-lightning/pull/318)
+
 ### Contributors
 
 This release contains contributions from (in alphabetical order):
```
Comment (chaeyeunpark) [inline .github/CHANGELOG.md:24]:
```suggestion
This release contains contributions from (in alphabetical order):
Lee James O'Riordan
```

**Review by chaeyeunpark — APPROVED** [review]
Looks good to me, Lee.

File: .github/CHANGELOG.md [hunk 2]
```diff
@@ -16,9 +16,13 @@
...
+Lee James O'Riordan
```
Comment (mlxd) [inline .github/CHANGELOG.md:25]:
```suggestion
```

**Comment by multiphaseCFD** [comment]
Thanks @mlxd ! Look fine to me.

**Review by multiphaseCFD — APPROVED** [review]

**Pull request merged**